### ISSUE TITLE
[RangeSlider] remove implementation of accessibility inputs

### ISSUE THREAD

**Issue by danrosenthal** [Shopify/polaris-react#839]
#784 implements the `DualThumb` range slider with text inputs for accessibility purposes. These should be removed in favor of documentation that describes or directly illustrates the use of alternative inputs for accessibility. This could either be illustrated in a second example, or described in the `Accessibility` section added in @dpersing's accessibility documentation work.

In addition to removing any markup and handlers for these elements, we will also want to remove the prop specified [here](https://github.com/Shopify/polaris-react/pull/784/files#diff-ee6d1f0846ba7ee065f302457e2963d6R37). 

|accessibility inputs shown in red|
|-|
|![screen_shot_2019-01-09_at_1_11_16_pm](https://user-images.githubusercontent.com/1403188/50919167-1112fc80-1410-11e9-8e4b-d103f5a5c5df.png)|

**Comment by dpersing** [comment]
> documentation that describes or directly illustrates the use of alternative inputs for accessibility.

@danrosenthal I'd strongly favor:

- Documentation that describes the accessibility differences between the single- and dual-thumb sliders out of the box (I can work on that today/this week)
- A best practice Playground/Storybook example showing the dual-thumb slider with text/number inputs.

**Comment by danrosenthal** [comment]
@dpersing I'm good with that!

**Comment by solonaarmstrong** [comment]
@dpersing FYI, Liz and I have a [WIP PR](https://github.com/Shopify/polaris-react/pull/843).
